docs/developer/slug-fields.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: Slug fields
+---
+
+## Introduction
+Slug is a short label for an object. Main reason for using slugs is ability to encorporate them in URLs. Translated names can contain all utf-8 signs, which could lead to URLs which could not be written for people with different keyboard layout.
```

Comment:
A slug is a short label that can explain an object or page's content. The main reason for using a slug is the ability to incorporate it to URLs. Translated names can contain all UTF-8 signs—this allow for URLs to be written by people with different keyboard layouts.